project/Version.scala

```diff
@@ -4,8 +4,8 @@ import sbt._
 object Version {
   val baseModelVersion = "0.1"
 
-  def getVersionString(baseDir: File) = {
-    def getLastModelCommitFromGit = { s"""git log -n 1 --pretty=format:%h ${baseDir.getAbsolutePath}""" !! }
```

Comment:
can you give a 1-2 sentence explanation to me about what these changes are and why they're needed?